src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -31,6 +32,17 @@ function getAmountModifiedFontSize(props: GetAmountModifiedFontSize) {
     : maxFontSize;
 }
 
+interface Lerp {
+  start: number;
+  end: number;
+  t: number;
+}
+
+// Linear Interpolation
```

Comment:
prob better to move it to `src/app/common/utils.ts`

Reply:
Done